installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,215 @@
+
+[id="installation-oc-mirror-workflow_{context}"]
+== High level workflow
+The following steps outline the high-level workflow on how to use the oc-mirror plugin to mirror images to a mirror registry:
+
+. Create an image set configuration file.
+. Mirror the image set to the mirror registry by using one of the following methods:
+** Mirror an image set directly to the mirror registry.
+** Mirror an image set to disk, transfer the image set to the target environment, then upload the image set to the target mirror registry.
+. Configure your cluster to use the resources generated by the oc-mirror plugin.
+. Repeat these steps to update your mirror registry as necessary.
+
+// About the oc-mirror plugin
+include::modules/oc-mirror-about.adoc[leveloffset=+1]
+
+// About V2
+include::modules/oc-mirror-v2-about.adoc[leveloffset=+2]
+
+// oc-mirror compatibility and support
+include::modules/oc-mirror-support.adoc[leveloffset=+1]
+
+// About the mirror registry
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For information about viewing the CRI-O logs to view the image source, see xref:../../installing/validating-an-installation.adoc#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+
+[id="prerequisites_installing-mirroring-disconnected"]
+== Prerequisites
+
+* You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2[Docker v2-2] in the location that will host the {product-title} cluster, such as Red Hat Quay.
++
+[NOTE]
+====
+If you use Red Hat Quay, you must use version 3.6 or later with the oc-mirror plugin. If you have an entitlement to Red Hat Quay, see the documentation on deploying Red Hat Quay link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/[for proof-of-concept purposes] or link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index[by using the Red Hat Quay Operator]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat Support.
+====
++
+If you do not already have an existing solution for a container image registry, subscribers of {product-title} are provided a link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/installing/disconnected-installation-mirroring#installing-mirroring-creating-registry[mirror registry for Red Hat OpenShift]. The _mirror registry for Red Hat OpenShift_ is included with your subscription and is a small-scale container registry that can be used to mirror the required container images of {product-title} in disconnected installations.
+
+* Save the file as `$XDG_RUNTIME_DIR/containers/auth.json`.
```

Comment:
I think this is misplaced, maybe from environment preparation?

Reply:
line 53?

Reply:
Yes its mentioned in line 69 below - so maybe delete this ?